docs/docs/self-hosting/deploy/docker-compose.yaml

```diff
@@ -1,41 +1,119 @@
 services:
-  zitadel:
-    restart: 'always'
-    networks:
-      - 'zitadel'
-    image: 'ghcr.io/zitadel/zitadel:latest'
-    command: 'start-from-init --masterkey "MasterkeyNeedsToHave32Characters" --tlsMode disabled'
+
+  db:
+    image: postgres:17-alpine
+    restart: unless-stopped
     environment:
-      ZITADEL_DATABASE_POSTGRES_HOST: db
-      ZITADEL_DATABASE_POSTGRES_PORT: 5432
-      ZITADEL_DATABASE_POSTGRES_DATABASE: zitadel
-      ZITADEL_DATABASE_POSTGRES_USER_USERNAME: zitadel
-      ZITADEL_DATABASE_POSTGRES_USER_PASSWORD: zitadel
-      ZITADEL_DATABASE_POSTGRES_USER_SSL_MODE: disable
-      ZITADEL_DATABASE_POSTGRES_ADMIN_USERNAME: postgres
-      ZITADEL_DATABASE_POSTGRES_ADMIN_PASSWORD: postgres
-      ZITADEL_DATABASE_POSTGRES_ADMIN_SSL_MODE: disable
-      ZITADEL_EXTERNALSECURE: false
+      - POSTGRES_USER=root
+      - POSTGRES_PASSWORD=postgres
+    networks:
+      - 'storage'
+    ports:
+      - '5432:5432'
```

Comment:
```suggestion
```

no need to bind a port. Postgres is reached of the internal network.

Using a port bind in this example would likely make it fail because devs (like me) might already have a posgreSQL running at the default port of 5432.